### ISSUE TITLE
[469b] UnrealEd Dragging actor 

### ISSUE THREAD

**Issue by NuLLut** [OldUnreal/UnrealTournamentPatches#466]
While moving Light actor the light moves at 50% of the camera speed when dragging with cntl left click

https://youtu.be/gGYM_DTnS7U

https://youtu.be/o4wO5y0csFY

Also more instance of same problem

https://youtu.be/GD7XGyyFLKE

And Un deleteable / un cuttable brush with the same drag issue 

https://youtu.be/zdSVeQXY7Qo

**Comment by SeriousBuggie** [comment]
Reason: on map exists duplicate actor.
So when you drag, position change apply twice or more.
Also you can delete actor only once. One copy deleted, another still in place and  look like undeletable.

How know: Select only one actor by click on it. Look at actor properties window. If there stated 2 or more - actor bugged.

How fix: select all such bugged actor and delete it. Save map. Load map.
Repeat this trick until exists bugged copies. If actor duplicate 3 times, you need do this twice, for example.

So reason of this behavior known, but reason of duplicate actors - unknown.

**Comment by SeriousBuggie** [comment]
> So reason of this behavior known, but reason of duplicate actors - unknown.

Reason for duplicate actors is known too now: #616

**Comment by SeriousBuggie** [comment]
Duplicate of #616

**Comment by stijn-volckaert** [comment]
We'll continue this discussion in #616
